packages/frontend/src/server/features/scaling/get-common-scaling-entry.ts

```diff
@@ -80,7 +122,36 @@ export function getCommonScalingEntry(
       })) ?? [],
     type: project.type,
     provider: project.display.provider,
-    hostChain: getHostChain(project),
+    hostChain: project.type === 'layer2' ? undefined : getHostChain(project),
     stage: project.stage ?? ({ stage: 'NotApplicable' } satisfies StageConfig),
+    filterable: {
+      isRollup: project.display.category.includes('Rollup'),
+      type: project.display.category,
+      stack: project.display.provider ?? 'No stack',
+      stage: getStage(project.stage),
+      purposes: project.display.purposes,
+      hostChain: project.type === 'layer2' ? 'Ethereum' : getHostChain(project),
+      daLayer:
+        getCurrentEntry(project.dataAvailability)?.layer.value ?? 'Unknown',
+      raas: getRaas(project.badges ?? []),
+    },
```

Comment:
mb let's move this to separate function that takes `project: Layer2 | Layer3 | Bridge` as an argument and returns filterable object

Reply:
A separate filterable for bridges is the way to go but needs a common entry first